packages/server/lib/routes.ts

```diff
@@ -152,8 +154,8 @@ if (AUTH_ENABLED) {
     web.route('/api/v1/account/signup/invite').get(rateLimiterMiddleware, authController.invitation.bind(authController));
     web.route('/api/v1/account/logout').post(rateLimiterMiddleware, authController.logout.bind(authController));
     web.route('/api/v1/account/signin').post(rateLimiterMiddleware, passport.authenticate('local'), signin);
-    web.route('/api/v1/account/forgot-password').put(rateLimiterMiddleware, authController.forgotPassword.bind(authController));
-    web.route('/api/v1/account/reset-password').put(rateLimiterMiddleware, authController.resetPassword.bind(authController));
+    web.route('/api/v1/account/forgot-password').post(rateLimiterMiddleware, postForgotPassword);
+    web.route('/api/v1/account/reset-password').put(rateLimiterMiddleware, putResetPassword);
```

Comment:
why the HTTP method at the beginning of the method? It reads a bit oddly and we don't do this currently anywhere else.

Reply:
we do it (or at least some), in orchestrator it's all like that and I used this pattern during log dev (so not all are named like this). It's handy to communicate expectation, also allows to dedup (i.e: putRecurring and postRecurring)

Reply:
Got it, thanks for clarifying.